internal/hook/item_hook_handler.go

```diff
@@ -38,35 +38,35 @@ import (
 type hookPhase string
 
 const (
-	hookPhasePre  hookPhase = "pre"
-	hookPhasePost hookPhase = "post"
+	PhasePre  hookPhase = "pre"
```

Comment:
should this be renamed to indicate it's for backup hooks?

Reply:
The main reason for the name change was to reduce stutter, so it wasn't `hook.HookPhase...`.

In terms of for backup or not, I'm not sure. Since restore won't currently have a phase at all, I'm not sure if it's necessary or not, but it sounds like you're in favor?

Reply:
Yes, I was hoping to make it explicit that there is no notion of a pre and a post hook for restore hooks.